components/ws-manager-bridge/src/bridge.ts

```diff
@@ -105,7 +105,7 @@ export class WorkspaceManagerBridge implements Disposable {
             // Still, listen to all updates, generate/derive new state and distribute it locally!
             startStatusUpdateHandler(false);
 
-            // emulate WorkspaceInstance updates for all Workspaces in the "preparing" phase in this cluster
+            // emulate WorkspaceInstance updates for all Workspaces in the "building" phase in this cluster
```

Comment:
Here we need to cover both phases for now 1) to be backwards compatible and 2) until control over imagebuild has shifted to `ws-manager`.

Reply:
Esp. we need to add "imagebuilding" to here: https://github.com/gitpod-io/gitpod/blob/8c6d938de09a601c49febcbfc3f84e71f70b2c51/components/ws-manager-bridge/src/preparing-update-emulator.ts#L47

Reply:
e2ca91814